client/package.json

```diff
@@ -11,11 +11,13 @@
   "dependencies": {
     "axios": "^0.21.0",
     "core-js": "^3.6.5",
+    "debug": "^4.2.0",
+    "package.json": "^2.0.1",
     "vue": "^2.6.11",
     "vue-material": "^1.0.0-beta-15",
+    "vue-observe-visibility": "^0.4.6",
     "vue-router": "^3.4.7",
-    "debug": "^4.2.0",
-    "vue-observe-visibility": "^0.4.6"
+    "vuelidate": "^0.7.6"
```

Comment:
Looked into how this works. Seems like we might need to learn more about it as a group.

Reply:
This wasn't supposed to make it into the pull request. Didn't end up using it. Will be removed in update.